.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules/
-.idea/
\ No newline at end of file
+.idea/
+
+package-lock.json
+yarn.lock
```

Comment:
Why not upload those?

Reply:
Because npm package not needed package-lock. package-lock is for the root project

Reply:
it is for CI but for package publish you want to see what happen to the users which install it